assembly/src/assembler/module_graph/mod.rs

```diff
@@ -416,6 +428,66 @@ impl ModuleGraph {
         Ok(())
     }
 
+    /// Resolves a [FullyQualifiedProcedureName] to its defining [Procedure].
+    pub fn find(
+        &self,
+        source_file: Option<Arc<SourceFile>>,
+        name: &FullyQualifiedProcedureName,
+    ) -> Result<GlobalProcedureIndex, AssemblyError> {
```

Comment:
This was removed in the previous PR or is this something new?

Reply:
Fully removed it as a result of removing `assemble_graph()`.